docroot/themes/contrib/civictheme/civictheme_library/components/01-atoms/field-message/field-message.scss

```diff
@@ -0,0 +1,42 @@
+//
+// Field description component.
+//
+
+@use 'sass:color';
+
+@mixin _civictheme-background-color-opacity($opacity ,$color) {
```

Comment:
please remove this in favour of using the rule directly through a global variable

```
background-color: $civictheme-field-message-light-background-color;
```

and then

```
$civictheme-field-message-light-background-color: color.adjust(civictheme-color('info'), -90);
```

https://sass-lang.com/documentation/modules/color#transparentize

Reply:
will need to define for each status message 
background-color: $civictheme-field-message-light-background-color;
ie: $civictheme-field-message-info-light-background-color;
$civictheme-field-message-error-light-background-color;

Reply:
@joshua-salsadigital 
`$civictheme-field-message-light-error-background-color` - the `theme` comes before the `type` in variable names

Reply:
yes, we need to define a variable for each type